### PR TITLE
Decreases the size of pills and patches

### PR DESCRIPTION
**Pull request by Spookuni**
## About The Pull Request
Decreases the maximum size of Pills to 15U and patches to 25U, decreases the reagent amount in most autogenerated pills to match the new maximum.

## Why It's Good For The Game
Chemistry has for a long time had something of an issue with a lack of meaningful opportunity to cost to many chemical setups - If you can get any chems in your system you can almost always get all the chems you want in your system. 50 units of reagent volume consumed instantly allows for enough healing reagents to heal every primary damage type except toxins to full multiple times over with room to spare for combat enhancers.
My hope is that by reducing the ease with which one can quickly and easily stack many types of healing chemicals, experimentation with different mixes and setups will become more valuable, rather than simply stacking every type of healing chemical into one pill and calling it a day.

## Changelog
:cl:
balance: Pills and Patches are now able to contain total less reagent volume due to an increase in required product safety metrics
balance: Standard complement pills have been reorganized to comply with new safety protocols and contain less reagents, maintenance pills are still unsafe as hell.
/:cl:

Github makes my brain hurt, so hopefully I haven't broken anything too important, please let me know if I have.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
This is a really bad change. Are you high?

**Comment by Burgermancoding** [comment]
Less of a chance for noobs to make something over the overdose threshold and accidentally die so I don't support this

**Comment by SmArtKar** [comment]
Don't think that this would impact game balance in a good way. Huge pills/patches are used pretty rarely right now already.

**Comment by D4C-420** [comment]
This won't do anything to benefit gameplay. People will just carry more pills with them and take more at once or they will just get a syringe and beaker for application. The benefit of using a pill is the lack of time for consumption and using a patch is that it can go through suits. The dosage amount was never the real issue.

**Comment by CRITAWAKETS** [comment]
Since when has chemstacking been a bad thing?

File: code/modules/reagents/chemistry/machinery/chem_master.dm [hunk 1]
```diff
@@ -307,9 +307,9 @@
 		var/list/style
 
 		if (item_type == "pill")
-			vol_each_max = min(50, vol_each_max)
+			vol_each_max = min(15, vol_each_max)
 		else if (item_type == "patch")
-			vol_each_max = min(40, vol_each_max)
+			vol_each_max = min(25, vol_each_max)
 		else if (item_type == "bottle")
 			vol_each_max = min(30, vol_each_max)
```
Comment (MrMelbert) [inline code/modules/reagents/chemistry/machinery/chem_master.dm:314]: No opinion on the adjustment itself, but these could/should be defines somewhere instead to make it easier to change or update (`MAX_PILL_VOLUME` or `MAX_PATCH_VOLUME`)

**Comment by Mothblocks** [comment]
I'm not convinced this is a good idea, I don't see a fundamentally large enough difference between taking multiple pills vs. taking one, or just keeping it in bottles/larger reagent containers and just drinking that.

CC @ExcessiveUseOfCobblestone

**Comment by stylemistake** [comment]
Multipatches are fun and balanced - very powerful, but they take more time to make, a lot more reagents go to waste and are less lasting due to dilution.

**Comment by ExcessiveUseOfCobblestone** [comment]
> Since when has chemstacking been a bad thing?

Since I started to champion this department. Its not a bad thing if you have sufficient downsides for each med so you make those tradeoffs when you stack, but it certainly is a nuisance if you can freely stack

1) chems of the same type
2) a bunch of different damage type meds (the omnipill)

I've considered this issue for a bit and havent really thought which route i wanted to pursue, but im not sure if just changing the volume of individual dosages, particularly ones you can take instantaneously anyways, is going to really tackle what I consider as the main issue.

That issue being that the resources you get at the start of the game, dispensers on top of plumbing constructors, allow you to practically make any and all chems without any real consideration towards what and how much you should be making (related to the "issue" of how much value 1u of reagent has).

**Comment by Mothblocks** [comment]
To jump off what Cobby is saying (or even just restate it in a simpler way), having multiple chems in one pill is fine is we can trust that the chemicals themselves have reasonable downsides.

**Comment by Spookuni** [comment]
> Multipatches are fun and balanced - very powerful, but they take more time to make, a lot more reagents go to waste and are less lasting due to dilution.

Multipatches and Multipills are still possible with the lowered numbers, they're just less potent and require more thought about what specifically you want to go in them, rather than essentially having enough space for everything in one pill that will solve every problem

As an example, a 50 unit pill can fit 8 units of both brute and burn C2 and C3 chems ( Libital, Sali, Oxan, and Aiuri) for a total of (assuming standard 75% purity) 3+4 = 7 brute per tick, and 2+4 = 6 burn per tick for (8/0.2) 40 ticks or ~80 seconds for a total healing of 280 brute and 240 burn respectively, with plenty of room to spare for more healing or other useful chemicals such as multiver, epinephrine, atropine, ephedrine, modafinil - you get the point. The idea of this PR isn't to nerf what chemistry can actually do at all - you can still make all of these chemicals and use them just fine, the point is to introduce opportunity cost to the application of the chemicals and make players think about what chemicals they want to introduce and when, rather than the classic old cure-all pills.

I will admit that the advent of fermichem has dramatically shifted chemistry as a sub-department towards macro spamming single chems and making the same thing every round anyway, but I'm working under the assumption that factories will eventually be a viable method of making chemicals without extreme investment again at some point in the future, at which point I expect we'll see cure all pills back in all of their glory.


> I'm not convinced this is a good idea, I don't see a fundamentally large enough difference between taking multiple pills vs. taking one, or just keeping it in bottles/larger reagent containers and just drinking that.

Drinking from a large beaker does allow for the same level of chemical intake as pills in theory, in practice it takes longer, requires a greater investment in what you're doing, and can be interrupted by someone stunning you while you're drinking or shoving you over, likewise with the requirement to use patches or multiple pills reduces the effectiveness of rapid chemical application and encourages planning as to what chems you want to consume and when you want to consume them.

I'm not married to the specific numbers I've used here, or to be frank, the idea itself, but I do think it would help cut down on some of the sillier heal-alls chemistry can (Once people figure out the macro for them with fermichem, lul) produce.

**Comment by AndroidMK2** [comment]
I don't see the need for those changes, current pills and patches size is good as it is.

**Comment by ExcessiveUseOfCobblestone** [comment]
I haven't bothered to consider how many iterations of a single normal chem should be able to fit in 1 of the bottles/pills/patches (more notably pills and patches since you have to take several delays per 5u in a bottle). Maybe a more important question is how much value should 1 of these have?

I've put in the design document that chem is definitely the cuts and bruises go to, so I would definitely want to make sure enough volume is in one pill to fulfill that concept adequately. I think it also makes sense to say that because they are the cuts and bruises department, the volume shouldn't be enough that they could 0-100 very easily which I think 50u total of dispenser chem combinations is very easily fulfillable.

Let me give this some more thought and I will get back to u.

**Comment by antropod** [comment]
Please, nobody asked for this. Pills and patches are good where they are.

Also, what about factory pills, I didn't see anything in code changes about them

**Comment by Spookuni** [comment]
While I solidly respect your opinion on chemistry balance Antropod, I'm currently not entirely convinced that this is actually a meaningful nerf to chemistry in its current state, outside of trying to kill chemstacking and/or insanely stupid megapills like 50u of libital healing for 1000 brute over 250 ticks.

The majority of simple single target pills I see in game already inhabit the general pill size area of 10-15u (encompassing a healing range of ~200-300 brute/burn with sali/oxan/pure libital and 135 - 202.5 burn with pure Aiuri). With some going up to 20u.

As I said, I'm not married to the specific numbers here, if 20u to cover the full range of normal non-insane chem pills or 25u to let people OD themselves on salicylic acid or oxandralone is preferable, I'm happy to adjust the numbers up. But up to 250 ticks at the standard 0.2 units/tick metabolism rate for most basic healing medicines is somewhat absurd.

My intent is not to make powerful chemical stacks that have a major impact on play impossible, and as several people have pointed out in this thread already, they wouldn't be if this PR was merged tomorrow. My intent is to make the uptake of extremely large dosages of mixed or pure chemicals somewhat harder and to introduce an element of opportunity cost to the different methods of chemical uptake.

As an example of still extant fast uptake methods, lethal injection syringes when self-used allow for the instant uptake of 50 units of chemicals just as current pills do, they are however a limited item that requires an enterprising chemist to negotiate with security in order to obtain, instead of being in literal infinite supply from roundstart. Likewise, bluespace syringes allow for extremely rapid chemical uptake when prepared properly, but are both time and material gated, reducing the number of them a chemist can hold at one time, and how quickly they can access them from roundstart, again, compared to a literal infinite spawner available from roundstart at no additional cost.

The amount of chemicals a chemist can put in themselves or others doesn't change with this PR, just how quickly and easily they can do it.

**Comment by antropod** [comment]
there are multiple methods to fill yourself with any amount of reagents in short time

bluespace syringes (they can instantly inject 60u)
or multiple IV drips (5u cycle each)
or foam (even after nerf you can easily fill yourself with 250-400 units of any mix roundstart if you use hercuri bottle and 10u foam)
or just take 2 pills instead of 1

just reducing pill size does not seem like a solution

**Comment by ExcessiveUseOfCobblestone** [comment]
see original message

**Pull request closed without merging**